backend/pkg/sqlmanager/postgres_sql-manager_integration_test.go

```diff
@@ -107,7 +107,7 @@ func (s *PostgresIntegrationTestSuite) Test_NewPooledSqlDb() {
 
 	conn, err := s.sqlmanager.NewPooledSqlDb(s.ctx, slog.Default(), s.mgmtconn)
 	requireNoConnErr(t, conn, err)
-	requireValidDatabase(t, s.ctx, conn, "postgres", "SELECT 1")
+	requireValidDatabase(t, s.ctx, conn, "pgx", "SELECT 1")
```

Comment:
huh, why did this just start failing?

Reply:
I merged in your changes to do a manual cli mysql sync test